src/libnyoci/nyoci-timer.c

```diff
@@ -178,7 +178,7 @@ nyoci_invalidate_timer(
 
 #if NYOCI_DEBUG_TIMERS || VERBOSE_DEBUG
 void
-nyoci_dump_all_timers(nyoci_t self) {
+static nyoci_dump_all_timers(nyoci_t self) {
```

Comment:
This should technically go on the previous line, but I wouldn't hold back this PR just for that.